.githooks/prepare-commit-msg

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+# Check for an empty commit message
+if [ -z "$2" ]
+then
+    echo "Empty commit message is not allowed."
```

Comment:
I tried this, 
seems like it would enforce usage of "git commit -m <files>"
rather than "git commit <files>"  which traditionally would spawn an editor to add the commit message.  
Just a matter of preference, but maybe worth mentioning it in after the commands to install the hook.

Reply:
Good point. I removed it. It seems like git catches the empty message regardless, so this was useless to begin with.